.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,69 @@
+name: SonarCloud
+on:
+  schedule:
+    - cron: "0 5 * * 2,4,6"
+#  pull_request:
```

Comment:
Not sure whether we want to use it as a quality gate on PRs since it may introduce excessive confusion w/o proper announcement and preparation (At first, I'd agreed on following code coverage, code duplication rules internally)

Reply:
Additionally, PRs from forks will not have access to the secret. I'd suggest adding and `if` block to the `sonarcloud` job instead.